src/ray/gcs/tables.h

```diff
@@ -26,15 +26,15 @@ class RedisContext;
 
 class AsyncGcsClient;
 
+/// \class Log
```

Comment:
It is a good idea to document at the top of a module what the expected/typical usage is.

For instance,
```
Example tables backed by Log:
    MyTable
Example tables backed by Table (single-entry datum):
   AnotherTable
```
so that readers are oriented.